tests/suites/test_suite_pk.data

```diff
@@ -38,6 +38,38 @@ EC(DSA) verify test vector #2 (bad)
 depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
 pk_ec_test_vec:MBEDTLS_PK_ECKEY:MBEDTLS_ECP_DP_SECP192R1:"046FDD3028FA94A863CD4F78DBFF8B3AA561FC6D9CCBBCA88E0AE6FA437F5415F957542D0717FF8B84562DAE99872EF841":"546869732073686F756C64206265207468652068617368206F662061206D6573736167652E00":"30350218185B2A7FB5CD9C9A8488B119B68B47D6EC833509CE9FA1FF021900FB7D259A744A2348BD45D241A39DC915B81CC2084100FA25":MBEDTLS_ERR_ECP_VERIFY_FAILED
 
+EC(DSA) verify test vector: good, bitlen(r) = 256
+depends_on:MBEDTLS_ECP_DP_SECP256R1_ENABLED
```

Comment:
Where are these test vectors taken from?

Reply:
You can read about the origin of these vectors here (courtesy of @mpg):
https://github.com/ARMmbed/mbedtls/pull/2159/commits/97929534825481b93d29e884c51c8e3131588319